### DISCUSSION TITLE
Working HP Elite Presenter Mouse

### DISCUSSION THREAD

**Discussion by transmissie** [gbin/Projecteur#220] [General]
Thank you for this fantastic software.
I have been using this with the Lenovo presenter since a long time and it is always a joy to use with your program for presentations at university.
I also have a HP Elite Presenter which until recently was not working because of bluetooth issues. I have it now working with opensuse 15.5  and also this presenter is really nice with your program.

The line for the devices.conf:
0x03f0, 0x464a, bt, Hp Elite Presenter

Thank you

**Comment by ExiMentor** [comment]
Hi, thanks for your comment!
I do have the same presenter, where is this file located?

**Reply by transmissie** [reply]
The devices.conf is in the top directory.
 
See also [compile time](https://github.com/jahnf/Projecteur?tab=readme-ov-file#compile-time)  for building the new version including your device in devices.conf

**Comment by ExiMentor** [comment]
Just wanted to say thank you! Very nice tool, worked fine for the last seminars. The only thing is, that my pointer falls to sleep after a while and it take some seconds to wake up.
